src/lib/errors.ts

```diff
@@ -45,6 +51,13 @@ export function UnitNotLoadedError(err: string[]): boolean {
 	return endsWith(err[0], 'not loaded.');
 }
 
+export function LockfileExistsError(err: ChildProcessError): boolean {
```

Comment:
Same thing here, this is better moved to where it makes more sense (probably the lockfile module)